packages/cli/src/functional/AgenticaStarter.ts

```diff
@@ -94,9 +94,11 @@ export namespace AgenticaStarter {
         }),
     },
     react: {
-      title: `React ${blueBright("Client Application")} (Currently not supported)`,
+      title: `React ${blueBright("Client Application")}`,
```

Comment:
I recommend to replace "Client Application" to "Not contains Agentica, only other server template client program"